src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.hpp

```diff
@@ -291,8 +276,8 @@ class SolidMechanicsLagrangianFEM : public SolverBase
   MPI_iCommData m_iComm;
   integer m_fixedStressUpdateThermoPoromechanicsFlag;
 
-  /// Rigid body modes
-  array1d< ParallelVector > m_rigidBodyModes;
+  /// Rigid body modes; TODO remove mutable hack
+  mutable array1d< ParallelVector > m_rigidBodyModes;
```

Comment:
This is now `mutable` because it's populated on demand in `createPreconditioner()` function which is `const`. This is isn't great, but I don't have a better solution for now.